imaging_install_MacOSX.sh

```diff
@@ -37,23 +37,26 @@ mridir=`pwd`
 #############################Create directories######################################
 #####################################################################################
 echo "Creating the data directories"
-  sudo -S su $USER -c "mkdir -p /data/$PROJ/data/"
-  sudo -S su $USER -c "mkdir -p /data/$PROJ/data/trashbin"          #holds mincs that didn't match protocol
-  sudo -S su $USER -c "mkdir -p /data/$PROJ/data/tarchive"          #holds tared dicom-folder
-  sudo -S su $USER -c "mkdir -p /data/$PROJ/data/pic"               #holds jpegs generated for the MRI-browser
-  sudo -S su $USER -c "mkdir -p /data/$PROJ/data/logs"              #holds logs from pipeline script
-  sudo -S su $USER -c "mkdir -p /data/$PROJ/data/jiv"               #holds JIVs used for JIV viewer
-  sudo -S su $USER -c "mkdir -p /data/$PROJ/data/assembly"          #holds the MINC files
-  sudo -S su $USER -c "mkdir -p /data/$PROJ/data/batch_output"      #contains the result of the SGE (queue
-  sudo -S su $USER -c "mkdir -p $mridir/dicom-archive/.loris_mri"
+  sudo -S su $USER -c "mkdir -m 2770 -p /data/$PROJ/data/"
```

Comment:
Say you have this scenario:

Using auto-launch, you upload a scan than was done in 2015 and you haven't upload any 2015 scans. MRProcessingUtilities will have to create /data/<proj>/data/tarchive/2015 (and when it does, gid will be set to www-data since weas re using the auto-launch). Once the scan is uploaded, you insert another one (done in 2015) but using the pipeline on the cmd line this time, run by lorisadmin. An error will occur since lorisadmin cannot create files in directory 2015.

Adding the sticky bit to /data/$PROJ/data will solve this. Makes sense ?